Update PVCDataSource to GA (#19318)
kubernetes/website: content/en/docs/concepts/storage/persistent-volumes.md

## Volume Cloning

> 

Reviewer: Should we keep the feature-state, but change it to GA/stable?

Reviewer 2: I'm fine with either approach. It's part of a long page; on the other hand, it's handy for readers to know that the feature is new.

Author: I had been following the pattern of other features that GA and just dropping it.  It seems like having it for initial GA is fine, but also it doesn't seem like past features had that.  

Let me know if you prefer it be added, otherwise I'll leave it out.

> Volume clone feature was added to support CSI Volume Plugins only. For details, see [volume cloning](/docs/concepts/storage/volume-pvc-datasource/).

Reviewer: Can you add a short blurb describing the high level concept?

> Volume clone feature was added to support CSI Volume Plugins only. For details, see [volume cloning](/docs/concepts/storage/volume-pvc-datasource/).

Reviewer 2: ```suggestion
[Volume cloning](/docs/concepts/storage/volume-pvc-datasource/) is only available for
CSI volume plugins.
```
